### ISSUE TITLE
Obtain final selector from filters

### ISSUE THREAD

**Issue by mazing** [aslagle/reactive-table#325]
I use the filters in my table, but I also need to print the selector outside my table.

So if all my filters are `name`, `age`, `height`, `weight`, etc., I am trying to get a selector

    selector = { name: '..', age: '..', height: '..', weight: '..' }

Is it possible to accomplish what I want? I have looked through the source, but it seems I have to do something with `getFilterStrings()`, `getFilterFields()`, but i am not quite sure how to get all the filters.

Can anyone point me in the right direction?

**Comment by aslagle** [comment]
getFilterQuery creates the actual selector that's used by the table, but it's not exposed outside the package. It's not meant to be used externally.

**Comment by mazing** [comment]
So it is not possible to get how the table is filtered without using `filter.get()` on each filter? :(

**Comment by aslagle** [comment]
No, sorry, not without forking the package and exposing more of the internal methods.

**Comment by mazing** [comment]
Okay :-(

Will it be implemented sometime? I love `reative-table` but to see how the table is filtered is crucial.

I have tried simply just adding a template helper returning

    return getFilterQuery(
      getFilterStrings(this.filters.get()),
      getFilterFields(this.filters.get(), this.fields),
      {enableRegex: this.enableRegex, filterOperator: this.filterOperator}
    );

but the it cannot find `this.filters`.
